lib/util.dart

```diff
@@ -564,7 +565,7 @@ class Utility {
       bool isOnboardingvalid = await OnBoardingApi().areRequiredFieldsValid();
       return shouldSkipOnboarding || isOnboardingvalid;
     } catch (e) {
-      print("Error checking onboarding status: $e");
+      print("Error checking onboarding status: ${e is TilerError?e.message:"Unknown error!"}");
```

Comment:
Same internationalization of string

Reply:
may I know why should we Internationalization prints and logs? and should I also Internationalization all print in onBoardingapi also?

Reply:
Good point we can ignore this one since it's just a print